apps/web/pages/reschedule/[uid].tsx

```diff
@@ -12,14 +12,11 @@ export default function Type() {
 }
 
 export async function getServerSideProps(context: GetServerSidePropsContext) {
-  const { uid: bookingId } = z
+  const { uid: bookingId, seatReferenceUid } = z
     .object({ uid: z.string(), seatReferenceUid: z.string().optional() })
     .parse(context.query);
-  let seatReferenceUid;
+
   const uid = await maybeGetBookingUidFromSeat(prisma, bookingId);
-  if (uid) {
-    seatReferenceUid = bookingId;
-  }
```

Comment:
I think there was a reason for this but can't remember right now. Will debug to see if anything happens.

Reply:
I think it was there because there is a time when rescheduling we don't share bookingUid with user only seatReferenceUid, so thats why we execute maybeGetBookingUidFromSeat and then we reassign values.